<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerOptionsContainer</class>
 <widget class="QWidget" name="AnswerOptionsContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="answerOptions"/>
   </item>
   <item>
    <widget class="QLabel" name="onHiddenAndEmptyListMessage">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Наведите курсор мыши на это сообщение, чтобы начать добавлять варианты ответов</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="addAnswerOptionLayout">
     <item>
      <widget class="QLineEdit" name="formulation">
       <property name="placeholderText">
        <string>Введите новый вариант ответа</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addAnswerOptionBtn">
         <property name="text">
          <string>Добавить вариант ответа</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
